baybe/recommenders/naive.py

```diff
@@ -1,4 +1,4 @@
-"""Naive hybrid recommender."""
+"""Naive hybrid space recommender."""
```

Comment:
Would prefer "Naive recommender for hybrid search spaces" as this is clearer.

Reply:
changed